### PR TITLE
Conditionals Concept Exercise Cleanup & Rename 

### PR DESCRIPTION
**Pull request by BethanyG**
Renamed exercise to `Meltdown Mitigation`.  Stub files reman as `conditionals.py` and `conditionals_test.py`.

- Rewrote `introduction.md`
- Rewrote `instructions.md`
- Created `hints.md`
- Created `exemplar.py`
- Checked and edited `conditionals_test.py`
- Added docstrings to `conditionals.py`
- Added a `/.meta/links.json` file
- Added `meltdown-mitigation` to track `config.json`
- Formatted `.meta/config.json` file
- Ran prettier

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -35,6 +35,14 @@
         "prerequisites": ["basics"],
         "status": "wip"
       },
+      {
+        "slug": "meltdown-mitigation",
+        "name": "Meltdown Mitigation",
+        "uuid": "50c88b36-e2e2-46e5-b6c4-bb7e714c375a",
```
Comment (sachsom95) [inline config.json:41]: Meltdown-mitigation is less boring than conditionals :)

File: exercises/concept/meltdown-mitigation/.docs/hints.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## General
+
+- The Python Docs on [Control Flow Tools][control flow tools] and the Real Python tutorial on [conditionals][real python conditionals] are great places to start.
+- The Python Docs on [Boolean Operations][boolean operations] can be a great refresher on `bools`, as can the Real Python tutorial on [booleans][python booleans].
+- The Python Docs on [Comparisons][comparisons] and [comparisons examples][python comparisons examples] can be a great refresher for comparisions.
+
+## 1. Check for criticality
+
+- Comparison operators and boolean operations can be _combined_.
```
Comment (sachsom95) [inline exercises/concept/meltdown-mitigation/.docs/hints.md:11]: This maybe difficult to understand for non native speakers. Maybe a one line example

File: exercises/concept/meltdown-mitigation/.docs/instructions.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+# Instructions
+
+In this exercise, we'll develop a simple control system for a nuclear reactor.
+
+For a reactor to produce the power it must be in a state of _criticality_.
+If the reactor is in a state less than criticality, it can become damaged.
+If the recator state goes beyond criticality, it can overload and result in a meltdown.
```
Comment (sachsom95) [inline exercises/concept/meltdown-mitigation/.docs/instructions.md:7]: nit: minor type for reactor

**Review by sachsom95 — CHANGES_REQUESTED** [review]
Thanks a lot for the substantial work done on the exercise. I could find only a minor typo and one line about combining operators and boolean which may be difficult for non native speakers to understand(commented).

File: exercises/concept/meltdown-mitigation/.docs/instructions.md (hunk 3 above)
Comment (BethanyG) [reply exercises/concept/meltdown-mitigation/.docs/instructions.md:7]: My spelling is often the most creative thing about me.  😉  😆

File: exercises/concept/meltdown-mitigation/.docs/introduction.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+# Conditionals
+
+The [conditionals][control flow tools] [`if`][if statement], `elif` (_a contraction of 'else and if'_) and `else` are used in Python to control the flow of execution and make decisions in a program.
+Unlike many other programming langauges, Python versions 3.9 and below do not offer a formal case-switch statement, using multiple `elif` statements to serve a similar purpose.
+Python 3.10 introduces a variant case-switch statement called `pattern matching`, which will be covered in another exercise.
+Conditional statements pair with expressions and objects that must resolve to `True` or `False` -- either by returning a `bool` directly, or by evaluating ["truthy" or "falsy"][truth value testing].
```
Comment (neenjaw) [inline exercises/concept/meltdown-mitigation/.docs/introduction.md:6]: This paragraph seems very familiar, I think I reviewed it with suggestions in another PR

File: exercises/concept/meltdown-mitigation/results.json [hunk 5]
```diff
@@ -0,0 +1,108 @@
+{
```
Comment (neenjaw) [inline exercises/concept/meltdown-mitigation/results.json:1]: Should this file be committed in this PR? Suggest deletion of same.

**Review by neenjaw — CHANGES_REQUESTED** [review]
I think the copy looks pretty good.  One minor comment about the one paragraph which I think I've seen before in another PR, if I suggested things there, they probably apply here unless you have already integrated them.

Reason for request change: I think that an extra result file from your test runner was included in this PR mistakenly

File: exercises/concept/meltdown-mitigation/.docs/introduction.md (hunk 4 above)
Comment (BethanyG) [reply exercises/concept/meltdown-mitigation/.docs/introduction.md:6]: You did.  And **_thank you_** for noting it.  You did have changes there, and I will make them here as well.  😄

File: exercises/concept/meltdown-mitigation/results.json (hunk 5 above)
Comment (BethanyG) [reply exercises/concept/meltdown-mitigation/results.json:1]: Ooof!  No.  This is a results file from testing and should not be here.  Thank you for catching that!!

**Review by neenjaw — APPROVED** [review]

**Comment by BethanyG** [comment]
This is an evil merge-confilct resolution mess.  Didn't know any other way of doing it, since the github instructions didn't work.

**Comment by BethanyG** [comment]
@iHiD  -- really not understanding what is going on here.  Thought the code owners changes weren't supposed to trigger needing approval from admins?!!?

**Review by neenjaw — APPROVED** [review]

**Pull request closed without merging**